python/ray/train/backend.py

```diff
@@ -51,12 +51,10 @@ class Backend(metaclass=Singleton):
     def on_start(self, worker_group: WorkerGroup,
                  backend_config: BackendConfig):
         """Logic for starting this backend."""
-        pass
```

Comment:
As we started talking about offline, let's spend some more time figuring out if we want to change this Singleton pattern altogether.

But assuming the proposed change remains the same, I still feel that an explicit `pass` statement improves readability. (Another option is to drop the `pass` but explicitly mention in the docstring that it's a no-op.)